src/assignment/views/AssignmentEdit.tsx

```diff
@@ -115,7 +116,7 @@ interface AssignmentEditProps extends RouteComponentProps {
 	loginState: LoginResponse | null;
 }
 
-// TODO: Test with a single useEffect
+// TODO: Replace with useReducer method.
```

Comment:
userState zit in de store
maar deze vars zitten niet in de store, denk je dat dit dan opgelost kan worden met een useReduce?

Reply:
Heb het over de useReducer React hook, niet Redux-reducer. In principe heeft dit iirc toch niets te maken met je global store. Is toch gewoon een hook om state "transitions" door te voeren in je component? Of zie ik dat verkeerd?

https://css-tricks.com/getting-to-know-the-usereducer-react-hook/

Reply:
ah, ziet er veelbelovend uit